infra/base-images/base-builder-new/install_swift.sh

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+#!/bin/bash
```

Comment:
Can you move this line to the top of the file and change it to "#!/bin/bash -eux"
This way errors will cause it to stop execution.

Reply:
Why is this file in base-builder-new rather than in base-builder-swift  ?

Reply:
if there are multi-lang projects, they'd inherit from their primary language but be able to call installations scripts of the other languages

Reply:
Excellent :-)